android/expoview/src/main/java/host/exp/exponent/Constants.java

```diff
@@ -71,7 +71,7 @@ public static void setSdkVersions(List<String> sdkVersions) {
   }
 
   static {
-    Set<String> abiVersions = new HashSet<>();
+    List<String> abiVersions = new ArrayList<>();
```

Comment:
i wonder why this was a `HashSet` to begin with 🤔  either way this seems correct